web/src/components/profileEdit/QuotaCardEdit.tsx

```diff
@@ -100,62 +181,152 @@ const QuotaCardEdit: React.FC<IQuotaCardEditProps> = (props) => {
-        <Label as="span" variant="errorLabel">
-          Not available due to a {isProvisioned ? 'Update' : 'Provision'} Request
-        </Label>
-      )}
-    </>
+
+      <Form
+        onSubmit={handleSubmit}
+        validate={(values) => {
+          const errors = {};
+          return errors;
+        }}
+      >
+        {(formProps) => {
+          const DisplayQuotaForm = Object.keys(QUOTAINFORMATIONCONSTANTS).map(
+            (element: any, index) => (
+              <Box key={index + QUOTAINFORMATIONCONSTANTS[element].title}>
+                <Text as="h3">{QUOTAINFORMATIONCONSTANTS[element].title}</Text>
+                <Flex flexDirection="column" paddingLeft="4">
+                  {QUOTAINFORMATIONCONSTANTS[element].options.map(
+                    (option: any, optionIndex: any) => (
+                      <Flex marginBottom="2" key={optionIndex + option.displayName}>
+                        <Label variant="adjacentLabel" m="auto" htmlFor="project-quota">
+                          {option.displayName}
+                        </Label>
+
+                        <Flex flex="1 1 auto" justifyContent="flex-end" name="project-quota">
+                          {option.name === 'QuotaSize' ? (
+                            // React-final-form onChange bug: https://github.com/final-form/react-final-form/issues/91
+                            <Field
+                              name="selectedSize"
+                              component={SelectInput}
+                              initialValue={`Current: ${option.value}`}
+                              validate={required}
+                            >
+                              <option> Current: {option.value} </option>
+                              {quotaOptions.length !== 0 &&
+                                quotaOptions.map((opt: any) => (
+                                  <option key={opt} value={opt}>
+                                    {opt.toUpperCase()}
+                                  </option>
+                                ))}
+                            </Field>
+                          ) : (
+                            <Label justifyContent="flex-end">
+                              <Text>{option.value}</Text>{' '}
+                              {applyingQuotaSpecs.length !== 0 && element !== 'Quota' && (
+                                <Flex marginLeft="1">
+                                  <Text> to {applyingQuotaSpecs[element][optionIndex]}</Text>
+                                </Flex>
+                              )}
+                            </Label>
+                          )}
+                        </Flex>
+                      </Flex>
+                    ),
+                  )}
+                </Flex>
+              </Box>
+            ),
+          );
+
+          const QuotaChangeComponent = (
+            <Flex
+              backgroundColor={theme.colors.grey}
+              paddingX={4}
+              paddingY={3}
```

Comment:
Similar to other notes, matching the style used throughout is definitely valuable.
```suggestion
              px={4}
              py={3}
```